plugins/woocommerce-admin/client/activity-panel/style.scss

```diff
@@ -204,16 +204,15 @@
 	}
 	transform: translateX(100%);
 	@include activity-panel-slide();
-	position: fixed;
+	position: absolute;
```

Comment:
This is meant to resolve a little bug I came across that caused the Activity Panel to be displayed incorrectly when the reminder bar was present.

Reply:
Nice catch @joshuatf . I just brought this up in `trunk`, and it shows the same behavior. The [original issue](https://github.com/woocommerce/woocommerce/issues/32161) doesn't express how the reminder bar should display with the new navigation enabled, but I'd say it's a good bet that we want it to _not_ span the entire page, similar to the classic navigation.

On first glance it might involve non-trivial changes to how this is structured to resolve, and it's not caused/impacted by this particular PR, so we may want to capture it in a separate issue.